dojo/tools/codechecker/parser.py

```diff
@@ -0,0 +1,128 @@
+
+        return list(items.values())
+
+
+def get_item(vuln):
+
+    description = 'Analyzer name: {}\n'.format(vuln['analyzer_name'])
+    description += 'Category name: {}\n'.format(vuln['category'])
+    description += 'Checker name: {}\n'.format(vuln['checker_name'])
+
+    if 'type' in vuln:
+        vuln_type = vuln.get('type', 'None')
+        if vuln_type != 'None':
+            description += 'Type: {}\n'.format(vuln_type)
+
+    if 'message' in vuln:
+        description += '{}\n'.format(vuln['message'])
+
+    location = vuln['file']
+    file_path = location['path'] if 'path' in location else None
+
+    if file_path:
+        description += 'File path: {}\n'.format(file_path)
+
+    line = vuln['line'] if 'line' in vuln else None
+    column = vuln['column'] if 'column' in vuln else None
+
+    if line is not None and column is not None:
+        description += 'Location in file: line {}, column {}\n'.format(line, column)
+
+    sast_source_line = line
+
+    sast_object = None
+
+    severity = get_mapped_severity(vuln.get('severity', 'UNSPECIFIED'))
+
+    mitigation = ''
+
+    references = ''
+
+    review_status = vuln.get('review_status', 'unreviewed')
+    verified = review_status == 'confirmed'  # bug confirmed by reviewer
+    risk_accepted = review_status == 'intentional'  # not confirmed, not a bug, there are some reasons to make this code in this manner
+    false_positive = review_status in ['false_positive', 'suppressed']  # this finding is false positive
+
+    hash = hashlib.sha256()
+    unique_id = vuln['report_hash'] + '.' + vuln['analyzer_result_file_path'] + description
+    hash.update(unique_id.encode())
+    unique_id_from_tool = hash.hexdigest()
+
+    title = ''
+    if 'checker_name' in vuln:
+        title = vuln['checker_name']
+    elif 'message' in vuln:
+        title = vuln['message']
+    else:
+        title = unique_id_from_tool
+
+    finding = Finding(title=title,
+                      description=description,
+                      severity=severity,
+                      mitigation=mitigation,
+                      unique_id_from_tool=unique_id_from_tool,
+                      references=references,
+                      file_path=file_path,
+                      line=line,
+                      verified=verified,
+                      risk_accepted=risk_accepted,
+                      false_p=false_positive,
+                      sast_source_object=sast_object,
+                      sast_sink_object=sast_object,
+                      sast_source_file_path=file_path,
+                      sast_source_line=sast_source_line,
+                      static_finding=True,
+                      dynamic_finding=False)
+
+    return finding
+
+
+def get_mapped_severity(severity):
+    switcher = {
+        'CRITICAL': 'Critical',
+        'HIGH': 'High',
+        'MEDIUM': 'Medium',
+        'LOW': 'Low',
+        'STYLE': 'Informational',
+        'UNSPECIFIED': 'Informational',
```

Comment:
```suggestion
        'STYLE': 'Info',
        'UNSPECIFIED': 'Info',
```
According to:
https://github.com/DefectDojo/django-DefectDojo/blob/4dfb3a43651846696823e2d3d7bd5ec485b54276/dojo/models.py#L46-L47

This is a NO-GO